testsuite/testcases/src/multi_region_network_test.rs

```diff
@@ -252,6 +266,18 @@ impl MultiRegionNetworkEmulationTest {
         }
     }
 
+    pub fn default_for_validator_count(num_validators: usize) -> Self {
+        if num_validators < 20 {
+            Self {
+                network_emulation_config: MultiRegionNetworkEmulationConfig::four_region(),
+            }
+        } else {
+            Self {
+                network_emulation_config: MultiRegionNetworkEmulationConfig::six_region(),
```

Comment:
This will force all tests with more than 20 validators to be 6 region, is that what we want? How about making this threshold above 100?

Reply:
Sure, made the change.